### PR TITLE
Integration test stage damage

### PR DESCRIPTION
**Pull request by rtle76**

### CODE REVIEW

**Review by rnugent3 — CHANGES_REQUESTED** [review]
This looks good. Please add unit test cases where variation in beginning damage depth results in variation in damage for a given structure.

**Comment by rnugent3** [comment]
Once you have those test cases set up, re-request review, and we'll get this branch merged.

File: fda-model-test/unittests/structures/DeterministicStructureShould.cs [hunk 1]
```diff
@@ -24,15 +22,16 @@ public class DeterministicStructureShould
...
         private static int impactAreaID = 55;
-        private static DeterministicStructure deterministicStructure = new DeterministicStructure(structureID, impactAreaID, sampledStructureParameters);
 
 
         [Theory]
-        [InlineData(100, 0, 0)]
-        [InlineData(104, 400, 200)]
-        [InlineData(108, 800, 400)]
-        public void DeterministicStructureShouldComputeDamageCorrectly(float waterSurfaceElevation, double expectedStructureDamage, double expectedContentDamage)
+        [InlineData(100, 0, 0, 999)]
```
[the remark this answers is not in the document]
Comment (rtle76) [reply fda-model-test/unittests/structures/DeterministicStructureShould.cs]: Requested changes made, re-requested pull. Let me know what you think

**Review by rnugent3 — APPROVED** [review]
Really good test cases. Great job, man. Keep it up.

**Pull request merged**